ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -35,39 +35,209 @@ define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
       }
     ]
   };
-  function reportError(msg) {
-    if (msg) {
-      console.error(msg);
-    }
+  function  assertAddTermCommon(assert, component) {
+    /** Common tests for Add terms, use in term and taxonomy components */
+    // check term title
+    var listOfLinks = React.addons.TestUtils.
+      scryRenderedDOMComponentsWithTag(
+        component,
+        'a'
+    );
+    assert.equal(listOfLinks.length, 2);
+    var linkHeader = listOfLinks[0].getDOMNode();
+    assert.equal(linkHeader.innerHTML, vocabulary.name);
+    //test items
+    var devItems = React.addons.TestUtils.
+      findRenderedDOMComponentWithClass(
+        component,
+        'panel-body'
+    );
+    var itemList = React.addons.TestUtils.
+      scryRenderedDOMComponentsWithTag(
+        devItems,
+        'li'
+    );
+    assert.equal(itemList.length, 2);
+    return devItems;
+  }
+
+  function assertAddVocabularyCommon(assert, component) {
+    /** Common tests for Add Vocabulary, use in term and taxonomy components */
+    var formNode = React.addons.TestUtils.
+      findRenderedDOMComponentWithClass(
+        component,
+        'form-horizontal'
+      );
+    assert.ok(formNode);
+    //test form submission
+    var inputNodes = React.addons.TestUtils.
+      scryRenderedDOMComponentsWithTag(
+        formNode,
+        'input'
+      );
+    assert.equal(inputNodes.length, 12);
+    inputNodes[0].value = "TestA";
+    inputNodes[1].value = "TestA";
+    inputNodes[10].checked = true;
```

Comment:
Please add a comment for what `inputNodes[10]` is or store it in a variable with a descriptive name.